feature/bgp/gracefulrestart/ate_tests/bgp_graceful_restart_test/bgp_graceful_restart_test.go

```diff
@@ -144,9 +147,14 @@ func bgpWithNbr(as uint32, nbrs []*bgpNeighbor) *telemetry.NetworkInstance_Proto
 	bgpgr.RestartTime = ygot.Uint16(grRestartTime)
 	bgpgr.StaleRoutesTime = ygot.Uint16(grStaleRouteTime)
 
+	pg := bgp.GetOrCreatePeerGroup(peerGrpName)
```

Comment:
Is a peer group required in this case?

Reply:
we have to define the peer group even if we aren't setting any policy because it's
 invalid OC for the neighbor to be part of a peer group that doesn't exist.